datahub-web-react/src/app/entity/dataProcessInstance/DataProcessInstanceEntity.tsx

```diff
@@ -181,13 +156,11 @@ export class DataProcessInstanceEntity implements Entity<DataProcessInstance> {
                 platformLogo={data?.dataPlatformInstance?.platform?.properties?.logoUrl}
                 owners={null}
                 globalTags={null}
-                // domain={data.domain?.domain}
                 dataProduct={getDataProduct(genericProperties?.dataProduct)}
                 externalUrl={data.properties?.externalUrl}
                 parentContainers={data.parentContainers}
-                parentEntities={parentEntities}
+                parentEntities={parentEntities as unknown as PreviewEntity[]}
```

Comment:
@asikowitz this was my attempt to fix the yarn type-check -- let me know if this makes sense